delivery_total_weight_from_packaging/__manifest__.py

```diff
@@ -1,14 +1,14 @@
 # Copyright 2020 Camptocamp SA
-# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
```

Comment:
Why this license change?

Reply:
Hi @sebalix, I changed the license because I think it is the same thing with this [PR](https://github.com/OCA/stock-logistics-warehouse/pull/1256).

Reply:
Ok, I checked internally and this seems OK. @simahawk are you agree with this change?

Reply:
I don't get the reference to https://github.com/OCA/stock-logistics-warehouse/pull/1256 :thinking: 
There's no license mix here AFAIS (on v13 at least).
Also, "hiding" the license change in the same commit IMO it's not that good.
If there's no special / good reason to make it AGPL I'd roll back this change

Reply:
Thanks @sebalix, @simahawk for the comments. We will roll back the change.

Reply:
This PR has been rebased and updated. Thanks.

Reply:
Runbot was red, I restarted it to check the logs